lib/billing/request.ts

```diff
@@ -65,6 +70,15 @@ export function createRequest(config: ConfigInterface) {
         isTest,
       });
       data = mutationResponse.data.appPurchaseOneTimeCreate;
+    } else if (billingConfig.interval === BillingInterval.Usage) {
```

Comment:
We could make this whole block a switch, probably easier to follow than if/elses.